<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyInterface</class>
 <widget class="QWidget" name="MyInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/oneimage/2.png);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    border-image:url(:/oneimage/closeback.png) 0 60 0 0 ;
}

QPushButton:hover
{
    border-image:url(:/oneimage/closeback.png) 0 30 0 30 ;
}

QPushButton:pressed
{
    border-image:url(:/oneimage/closeback.png) 0 0 0 60 ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    border-image:url(:/oneimage/1.png) 0 60 0 0 ;
}

QPushButton:hover
{
    border-image:url(:/oneimage/1.png) 0 30 0 30 ;
}

QPushButton:pressed
{
    border-image:url(:/oneimage/1.png) 0 0 0 60 ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;QQ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#f9f9f9;&quot;&gt;QQ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="MyIconLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;id号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="MyNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;用户名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    border-image:url(:/oneimage/3.png) 0 60 0 0 ;
}

QPushButton:hover
{
    border-image:url(:/oneimage/3.png) 0 30 0 30 ;
}

QPushButton:pressed
{
    border-image:url(:/oneimage/3.png) 0 0 0 60 ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>241</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MyInterface</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>MyInterface</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
